src/main/java/org/jenkinsci/plugins/workflow/job/console/WorkflowRunConsoleNote.java

```diff
@@ -24,23 +24,22 @@
 
 package org.jenkinsci.plugins.workflow.job.console;
 
-import org.jenkinsci.plugins.workflow.job.WorkflowRun;
-
 import hudson.MarkupText;
 import hudson.console.ConsoleAnnotator;
 import hudson.console.ConsoleNote;
 import hudson.model.Run;
+import hudson.model.TaskListener;
+import java.io.IOException;
 
 /**
  * Console note for Workflow metadata specific messages.
- * See {@link WorkflowConsoleLogger} for more information.
  */
 public class WorkflowRunConsoleNote extends ConsoleNote<Run<?, ?>> {
 
     /**
      * Prefix used in metadata lines.
      */
-    public static final String CONSOLE_NOTE_PREFIX = "[Pipeline] ";
+    private static final String CONSOLE_NOTE_PREFIX = "[Pipeline] ";
```

Comment:
Just minimizing access of things which were never really supposed to be `public`.